.github/workflows/microservices.yml

```diff
@@ -10,10 +10,11 @@ jobs:
       with:
         path: DCSA-TNT
         submodules: recursive
-    - name: Set up Java JDK
-      uses: actions/setup-java@v1
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
       with:
-        java-version: 11
+        java-version: 17
+        distribution: 'adopt'
```

Comment:
adopt is no longer being updated, please use temurin instead

Reply:
done